helm/cluster/values.schema.json

```diff
@@ -1844,6 +1859,19 @@
                                     "$ref": "#/$defs/appEphemeral"
                                 }
                             }
+                        },
+                        "offlineTesting": {
+                            "type": "object",
+                            "title": "Offline testing",
+                            "description": "Configure how Helm template rendering behaves when it is running during testing (e.g. locally or in the CI) and without connection to cluster.",
+                            "additionalProperties": false,
+                            "properties": {
+                                "renderWithoutReleaseResource": {
```

Comment:
A flag that enables rendering all manifests without Release CR existing. This is `false` by default, and it's set to true only in CI which is just rendering manifests without creating clusters.